catalog/tests/dags/providers/provider_api_scripts/test_europeana.py

```diff
@@ -100,9 +104,93 @@ def test_get_batch_data_gets_items_property(ingester):
 
 
 def test_get_image_list_with_realistic_response(ingester):
-    response_json = _get_resource_json("europeana_example.json")
-    record_count = ingester.process_batch(response_json["items"])
-    assert record_count == len(response_json["items"])
+    image_store = ImageStore(provider=prov.EUROPEANA_DEFAULT_PROVIDER)
+    ingester.media_stores = {"image": image_store}
+    batch_json = _get_resource_json("europeana_example.json")
+    object_json = {}
+    with patch.object(
+        ingester, "_get_additional_item_data", return_value=object_json
+    ) as item_call:
+        with patch.object(image_store, "add_item"):
+            record_count = ingester.process_batch(batch_json["items"])
+            assert item_call.call_count == len(batch_json["items"])
+            assert record_count == len(batch_json["items"])
+
+
+ITEM_HAPPY_RESPONSE = _get_resource_json("item_full.json")
+ITEM_NOT_1ST_RESPONSE = _get_resource_json("item_not_first_webresource.json")
+ITEM_HAPPY_WEBRESOURCE = {
```

Comment:
This looks like it could be extracted from the `item_full.json` file. Could it be the same for `ITEM_NOT_1ST_WEBRESOURCE`?